packages/office-ui-fabric-react/src/components/Modal/examples/Modal.Basic.Example.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../../common/common';
+
+:global {
+  .ms-modalMainOverride {
```

Comment:
Small nit, change this class name to include "Example" so that users don't assume it comes with the component.